promisasu/Portal
# Add Security Headers

**Taiga User Story**
None

**Taiga Task(s)**
None

**What did you change?**
Enabled HSTS, XFrame, XSS-protection, Download Options and No Sniff security headers on Dashboard and API.
Enabled CORS by default for API.

**How can we test?**
`npm start`
Dashboard and API should load as usual.
Inspecting Response headers should show:
![response headers](https://cloud.githubusercontent.com/assets/3107513/14938037/a5072e76-0ecc-11e6-956d-1ac6c640273f.PNG)
